### PR TITLE
Validation of min/max dates for queue

### PR DESCRIPTION
**Pull request by kpbacode**
This PR address fallowing issue:

- https://github.com/killbill/killbill-admin-ui/issues/272

### CODE REVIEW

File: app/controllers/kaui/queues_controller.rb [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
     rescue KillBillClient::API::NotFound
       # If TestResource is not bound, then clock has not been manipulated, we can default to NOW
-      @now = DateTime.now.in_time_zone("UTC")
+      @now = DateTime.now.in_time_zone('UTC')
     end
 
-    min_date = params[:min_date] || '1970-01-01'
-    max_date = params[:max_date] || '2100-01-01'
+    min_date = Time.parse(params[:min_date]).iso8601 unless params[:min_date].blank? || '1970-01-01'
+    max_date = Time.parse(params[:max_date]).iso8601 unless params[:min_date].blank? || '2019-10-12T03:00:00+05:30'
```
Comment (pierre) [inline app/controllers/kaui/queues_controller.rb:37]: Should be `params[: max_date]`.

Also, maybe we should just default to `now` for the max date?

**Review by pierre — CHANGES_REQUESTED** [review]
Just a typo.

File: app/controllers/kaui/queues_controller.rb (hunk 1 above)
Comment (kpbacode) [reply app/controllers/kaui/queues_controller.rb]: > Also, maybe we should just default to `now` for the max date?

PR updated.

Comment (kpbacode) [reply app/controllers/kaui/queues_controller.rb]: Sorry for typo

Comment (kpbacode) [reply app/controllers/kaui/queues_controller.rb]: Added tests and url fix

**Review by pierre — APPROVED** [review]

**Pull request merged**